integration/README.md

```diff
@@ -0,0 +1,47 @@
+# Quick Base Craft Demo | Senior Integration Specialist
```

Comment:
Let's just say "integration specialist" here. We'll probably use the same demo for other levels as well if we have more than one person in this position.

Reply:
true, thanks